tests/php/test-class-plugin-base.php

```diff
@@ -44,6 +44,16 @@ public function test_locate_plugin() {
 	}
 
 	/**
+	 * Test relative_path.
+	 *
+	 * @see Plugin_Base::relative_path()
+	 */
+	public function test_relative_path() {
+		$this->assertEquals( 'plugins/foo-bar', $this->plugin->relative_path( '/srv/www/wordpress-develop/src/wp-content/plugins/foo-bar', 'wp-content', '/' ) );
```

Comment:
This test is just checking that the relative_path method strips out the path before the starting point, so it's not dependant on VVV for it to work. I just used the VVV path for the test since it was the one that I use. Want me to add additional paths?